### PR TITLE
docs: add samples from dialogflow/cloud-client

### PR DESCRIPTION
**Pull request by busunkim96**
Samples from https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/dialogflow/cloud-client.

Also includes template updates for samples.

### CODE REVIEW

**Comment by busunkim96** [comment]
Please ignore the failing 'docs-presubmit' build. That will be resolved with further template updates in the microgenerator PR.

**Review by munkhuushmgl — APPROVED** [review]

**Comment by busunkim96** [comment]
@dinagraves Friendly ping

File: samples/snippets/README.rst [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+.. _Authentication Getting Started Guide:
+    https://cloud.google.com/docs/authentication/getting-started
+
+
+
+
+Install Dependencies
+++++++++++++++++++++
+
+#. Clone python-docs-samples and change directory to the sample directory you want to use.
```
Comment (busunkim96) [inline samples/snippets/README.rst:40]: Note: These repository references are incorrect now. The template issue is tracked in https://github.com/googleapis/synthtool/issues/867.

**Review by engelke — APPROVED** [review]
LGTM

**Pull request merged**